py/torch_tensorrt/dynamo/runtime/_MutableTorchTensorRTModule.py

```diff
@@ -475,7 +475,9 @@ def _process_kwarg_inputs(inputs: Any) -> Any:
             + "Allowed input types: {torch_tensorrt.Input, torch.Tensor, list, tuple, dict}"
         )
 
-    def forward(self, *args: Any, **kwargs: Any) -> Any:
+    # Due to https://github.com/pytorch/pytorch/issues/157183, we cannot use forward as a workaround.
+    # This is a temporary fix.
+    def call_forward(self, *args: Any, **kwargs: Any) -> Any:
```

Comment:
Just make forward call __call__ and put in a rel note instead of changing the api